web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -11,48 +11,38 @@
 	export let curve = false;
 	export let shadow = false;
 	export let circle = false;
-	let loading = true;
 
-	let imageElement: HTMLImageElement;
+	let complete = false;
 
-	onMount(() => {
-		if (thumbhash) {
-			lazyLoad(imageElement, {
-				hash: thumbhash,
-				hashType: 'thumbhash'
-			});
-		}
-	});
+	const decodeBase64 = (s: string) => Uint8Array.from(atob(s), (c) => c.charCodeAt(0));
```

Comment:
Have you tested this with SSR? `atob` can't be used by Node.js if I remember correctly

Reply:
I think `atob` is still in NodeJS, but considered deprecated. We had a chat in Discord about how to use `Buffer`, but couldn't get it to work. If you have any insight, I'd be grateful.

```ts
import { Buffer } from 'buffer';

Buffer.from('', 'base64');
```

Reply:
Fixed, I just needed to rebuild the containers for it to work.